subcommand/controller/command.go

```diff
@@ -122,19 +122,28 @@ func (c *Command) Run(args []string) int {
 	if err = (&controllers.ServiceDefaultsController{
 		ConfigEntryController: configEntryReconciler,
 		Client:                mgr.GetClient(),
-		Log:                   ctrl.Log.WithName("controllers").WithName("ServiceDefaults"),
+		Log:                   ctrl.Log.WithName("controller").WithName("servicedefaults"),
 		Scheme:                mgr.GetScheme(),
 	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "ServiceDefaults")
+		setupLog.Error(err, "unable to create controller", "controller", "servicedefaults")
 		return 1
 	}
 	if err = (&controllers.ServiceResolverController{
 		ConfigEntryController: configEntryReconciler,
 		Client:                mgr.GetClient(),
-		Log:                   ctrl.Log.WithName("controllers").WithName("ServiceResolver"),
+		Log:                   ctrl.Log.WithName("controller").WithName("serviceresolver"),
 		Scheme:                mgr.GetScheme(),
 	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "ServiceResolver")
+		setupLog.Error(err, "unable to create controller", "controller", "serviceresolver")
+		return 1
+	}
+	if err = (&controllers.ServiceRouterController{
+		ConfigEntryController: configEntryReconciler,
+		Client:                mgr.GetClient(),
+		Log:                   ctrl.Log.WithName("controller").WithName("servicerouter"),
```

Comment:
you can pull `servicerouter` into the common package which should be available once you rebase `crd-controller-base`
ProxyDefaults potentially still says "controllers" as well